storage/options.go

```diff
@@ -63,6 +64,12 @@ const (
 
 	// defaultRepairEnabled enables repair by default
 	defaultRepairEnabled = true
+
+	// defaultErrorWindowForLoad is the default error window for evaluating server load
+	defaultErrorWindowForLoad = 10 * time.Second
+
+	// defaultErrorThresholdForLoad is the default error threshold for considering server overloaded
+	defaultErrorThresholdForLoad = 100
```

Comment:
100 in the past 10 seconds seems a little slow to react considering we can easily be doing 150,000 writes per second.  Can we do 100 every 1 second perhaps?